src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -118,14 +118,19 @@
             "workdir": "Verzeichnis",
             "allowOnlineEditor": {
                 "title": "Online-Editor erlauben",
-                "alert": "Es muss mindestens eine Option (Offline-IDE oder Online-Editor) ausgewählt sein"
+                "alert": "Es muss mindestens eine Option (Offline-IDE, Online-Editor oder Online-IDE) ausgewählt sein"
             },
             "onlineEditor": "Online",
             "allowOfflineIde": {
                 "title": "Offline-IDE erlauben",
-                "alert": "Es muss mindestens eine Option (Offline-IDE oder Online-Editor) ausgewählt sein"
+                "alert": "Es muss mindestens eine Option (Offline-IDE, Online-Editor oder Online-IDE) ausgewählt sein"
             },
             "offlineIde": "IDE",
+            "allowOnlineIde": {
+                "title": "Online-IDE erlauben",
+                "alert": "Es muss mindestens eine Option (Offline-IDE, Online-Editor oder Online-IDE) ausgewählt sein"
```

Comment:
Why is this now duplicated three times? I think there should be a better solution

Reply:
It was already duplicated twice, I feel like breaking the consistency with `topic`.`alert` would be worse than a third duplication.